src/Facets/HopFacetPacked.sol

```diff
@@ -20,7 +48,23 @@ contract HopFacetPacked is ILiFi, TransferrableOwnership {
 
     /// @notice Initialize the contract.
     /// @param _owner The contract owner to approve tokens.
-    constructor(address _owner) TransferrableOwnership(_owner) {}
+    constructor(
+        address _owner,
+        address _wrapper
+    ) TransferrableOwnership(_owner) {
+        l2CanonicalToken = _wrapper != address(0)
```

Comment:
If I understand correctly, we allow this facet to be initiated with  `_wrapper == address(0)` since on L1 (=ETH) there is no wrapper, right?

Maybe we could add a check that allows _wrapper to be only address(0) on non-ETH networks (to make sure that the facet is functional.